src/integrate/integrate.ts

```diff
@@ -42,14 +44,15 @@ const syncToAdmina = async (source: Source, inputs: Record<string, string>) => {
       console.log('Getting Azure AD data...')
       const azureAdData = await AzureAdSource.fetchApps(inputs)
       console.log('Registering custom service...')
-      await Promise.all(
-        azureAdData.map(async (app: AzureAdSource.AppInfo) => {
+      await PromisePool
```

Comment:
PromisePool won't throw error, it collects all errors, so I think we manually have to handle it.

Reply:
I fixed it 👏